java/test/jmri/jmris/AbstractThrottleServerTestBase.java

```diff
@@ -61,12 +59,10 @@ public void sendStatusTest(){
      */
     abstract public void confirmThrottleStatusSent();
 
-    @BeforeEach
     // derived classes must configure the ThrottleServer variable (ats)
     // and should also install a throttle manager.
     abstract public void setUp();
 
-    @AfterEach
     public void postTestReset(){
```

Comment:
Why did you removed this annotation? Does sub classes override it and have the annotation?

Reply:
Right.  Apparently, JU5 wants the annotations to be on the concrete class.